docs/vocabulary.md

```diff
@@ -388,6 +389,75 @@ pipeline in corresponding documentation.
+
+"Binary Source" refers to executables that are inputs to the pipeline, such as: Executable Software Dependencies, Container Images, Virtual Machine Images
+
+"Generated Software" refers to human-readable files that are outputs of the pipeline, such as: Compiled Software, Generated Configuration Files, Compiled Documentation, Baseline/Composition Information generated by the build. 
+
+"Generated Binaries" refers to executables that are outputs of the pipeline, such as: Executable Software, Container Images, Virtual Machine Images
+
+#### Build Stage
+* Semantics: Set up the build workspace and provision build resources. Download, retrieve, assemble, and/or compile software into an executable and testable format. Download, retrieve, assemble, and/or compile documentation into a consumable format.
+* Aliases: Compile
+* Inputs: Software Source, Binary Source
+* Outputs: Generated Software, Generated Binaries
+* Other Results and Side Effects: The build may also capture point-in-time details about the build environment and tools (compiler versions, package manager versions, task container versions in tekton, OS versions etc.) for anyone who wants to rebuild or understand a build.
+
+#### Test Stage
+* Semantics: Test, scan, verify, and lint software and documentation.
+* Aliases: Verify
+* Inputs: Software Source, Binary Source, Generated Software, Generated Binaries
+* Outputs: Test/Lint/Scan Records, Test/Lint/Scan Reports, Test/Lint/Scan Coverage Reports
+* Other Results and Side Effects: N/A
+
+#### Release Stage
+* Semantics: Package, version, sign, and publish software artifacts and documentation.
+* Aliases: Deliver, Publish
+* Inputs: Software Source, Binary Source, Generated Software, Generated Binaries
+* Outputs: Release Records, Release Reports
+* Other Results and Side Effects: Documentation and Software: packaged, signed, and published to a repository.
+
+#### Deploy Stage
+* Semantics: Deploy software artificats and documentation to any environment other than the build environment. Verify successful deployment.
+* Aliases: Install
+* Inputs: Software Source, Binary Source, Generated Software, Generated Binaries
+* Outputs: Deployment Records, Deployment Reports, Secrets to access deployed resources
+* Other Results and Side Effects: Documentation hosted in a review, staging or production environment. Software running in a review, staging or production environment.
+
+#### Maintain Stage
+* Semantics: Automatically update or upgrade previously built software. May invoke some or all of the previous stages.
+* Aliases: Update, Upgrade
+* Inputs: See Above
+* Outputs: See Above
+* Other Results and Side Effects: Updated documentation and software built, tested, released and deployed. This stage can also update source code repositories, for example, with new Declared Dependencies or Baseline/Composition Information. 
+
+#### Any Stage
```

Comment:
Could this be moved first and the format changed to make it more clear it describes general information and not another stage?

Reply:
OK. Done.